config.js

```diff
@@ -79,30 +57,17 @@ exports.updateFromArgv = function(argv) {
     if (argv.logfile && typeof argv.logfile === "boolean") {
         exports.logfile = `gtp2ogs_logfile_${new Date().toISOString()}`;
     }
-    for (const k of ["timeout", "startupbuffer"]) {
-        if (argv[k]) {
-            // Convert some times to microseconds once here so
-            // we don't need to do it each time it is used later.
-            exports[k] = argv[k] * 1000;
-        }
-    }
+    exportMicroseconds("timeout", argv);
```

Comment:
`exportMicrosecondsAsSeconds` is a more accurate name.

Reply:
oh actually it seems we were all mistaken
https://github.com/online-go/gtp2ogs/blob/devel/docs/OPTIONS-LIST.md#timeout

this exports time in seconds input as milliseconds output
so i guess `exportSecondsAsMilliseconds` is the correct name?